client/src/components/brushableHistogram/index.tsx

```diff
@@ -55,6 +55,7 @@ const HEIGHT_MINI = 15 - MARGIN_MINI.TOP - MARGIN_MINI.BOTTOM;
     isColorAccessor:
       (state as any).colors.colorAccessor === field &&
       (state as any).colors.colorMode !== "color by categorical metadata",
+    userDefinedGenes: (state as any).controls.userDefinedGenes,
```

Comment:
Instead of adding additional redux state to the component and causing re-renders on every mutation on `userDefinedGenes`, I wonder if it would be more performant to instead do this as within a reducer? This isn't an ask, but rather an ask. Might be worth writing up a ticket

Reply:
Consolidated in the latest version.